src/com/projectkorra/projectkorra/airbending/flight/FlightMultiAbility.java

```diff
@@ -82,9 +89,23 @@ public FlightMultiAbility(Player player) {
 			return;
 		}
 		
+		CoreAbility abil = null;
+		if (hasAbility(player, AirSpout.class)) {
+			abil = getAbility(player, AirSpout.class);
+		} else if (hasAbility(player, WaterSpout.class)) {
+			abil = getAbility(player, WaterSpout.class);
+		} else if (hasAbility(player, FireJet.class)) {
+			abil = getAbility(player, FireJet.class);
+		} else if (hasAbility(player, AirScooter.class)) {
+			abil = getAbility(player, AirScooter.class);
+		}
+		
+		if (abil != null) {
+			abil.remove();
+		}
+		
 		MultiAbilityManager.bindMultiAbility(player, "Flight");
-		hadFly = player.isFlying();
-		canFly = player.getAllowFlight();
+		flight = ProjectKorra.flightHandler.createInstance(player, "FlightMultiAbility");
```

Comment:
Use getName() or store a final String with the value of "FlightMultiAbility" instead

Reply:
Doesn't make much difference what identifier I use tbh, but sure I'll change it

Reply:
Yeah, I just think it's more aesthetically pleasing, and since it's being used in multiple locations it makes sense to do so imo